blocks/extensions/plot.py

```diff
@@ -136,14 +132,86 @@ def preexec_fn():
             logger.info('Plotting server PID: {}'.format(self.sub.pid))
         else:
             self.sub = None
-        output_server(self.document, url=self.server_url)
 
-    def __getstate__(self):
-        state = self.__dict__.copy()
-        state['sub'] = None
-        return state
+        self.session = Session(
+            root_url=self.server_url,
+            load_from_config=False)
+        output_server(self.document, session=self.session)
+
+
+class PlotHistogram(SimpleExtension):
+
+    def __init__(self, document, channels, server_url='default', **kwargs):
+        """ Live plot an histogram of mean and std dev of the provided channels
```

Comment:
Apparently pep257 is not complaining for the length of the docstring lines (running it locally). I will check Travis output when I am done with the rest.